src/nvim/ops.c

```diff
@@ -3893,7 +3893,8 @@ void ex_display(exarg_T *eap)
             msg_puts_attr("^J", attr);
             n -= 2;
           }
-          for (p = yb->y_array[j]; *p && (n -= ptr2cells(p)) >= 0; p++) {
+          for (p = yb->y_array[j]; *p && n >= 0; p++) {
+            n -= ptr2cells(p);
```

Comment:
doesn't this change the meaning of the code?

Reply:
Ughhhh, you're right. I did make a small test to see if it was correct but it was a bad test it turns out. Thanks for the catch.

Reply:
That PVS warning was for me :)